contracts/OptionPairFactory.sol

```diff
@@ -35,6 +35,7 @@ contract OptionPairFactory is IOptionPairFactory {
     mapping(address => address) public getTreasury;
     mapping(address => address) public getCollateral;
     address[] public options;
+    address[] public obligations;
```

Comment:
I think it might make sense to move both of these to Subgraph to save gas, what do you think @danhper?

Reply:
I was thinking the same thing.
Do you think that keeping a list of obligations (vs option) would make more sense? It's easy to get option from obligation but not vice-versa.

Reply:
Just realised that the obligation address could be fetched easily through option factory so nevermind!